lib/live_admin/router.ex

```diff
@@ -86,7 +83,12 @@ defmodule LiveAdmin.Router do
     end
   end
 
-  def build_session(conn, resources, title) do
+  def build_session(conn, opts) do
+    LiveAdmin.put_config(opts)
```

Comment:
this is probably not the best place to do put_config, as we it only needs to be call once rather than for each session, but also shouldn't be called at compile time